README.md

```diff
@@ -28,7 +28,7 @@ SkyAPM C#/.NET instrument agent
 > MyGet feed URL https://www.myget.org/F/skyapm-dotnet/api/v3/index.json
 
 # Supported
-- This project currently supports apps targeting netcoreapp2.0/netframework4.6.1 or higher.
+- This project currently supports apps targeting netcoreapp3.1、net5.0、net6.0 or higher.
```

Comment:
As this platform level upgrade, should we update the dotnet agent to 2.x for the next release?
This seems a big break change.

Reply:
I agree. I will also support the skywalking log protocol in the agent, and then release version 2.0.